README.md

```diff
@@ -147,6 +147,22 @@ Once that's done, you can run the manager and API services:
     $ sudo esi-leap-api
 ```
 
+### Installation using Containerization
+
+By encapsulating the ESI Leap into a container, all the necessary dependencies, configurations and services are bundled into a single package.
```

Comment:
It's probably worth mentioning that esi-leap.conf needs to be generated, and needs to be present in the directory that you're running `podman build` from.

Reply:
Updated readme and pushed the changes @tzumainn